src/commons/bgp/report.rs

```diff
@@ -112,34 +112,34 @@ impl From<BgpAnalysisSuggestion> for RoaDefinitionUpdates {
             suggestion.redundant,
         );
 
-        let mut added: HashSet<RoaDefinition> = HashSet::new();
-        let mut removed: HashSet<RoaDefinition> = HashSet::new();
+        let mut added: Vec<RoaDefinition> = vec![];
```

Comment:
Is this change from HashSet to Vec about stable ordering for testing? Does it matter that this type now seems to permit duplicates which would not have been possible before with the HashSet?